charts/corda-dev-prereqs/templates/kafka/kafka-statefulset.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     spec:
       containers:
         - name: "kafka"
-          image: "{{ .Values.imageRegistry }}/confluentinc/cp-kafka:7.3.3"
+          image: "{{ .Values.imageRegistry }}/confluentinc/cp-kafka:7.5.3"
```

Comment:
Note that this is still Kafka 3.5. Confluent Platform does not yet support 3.6.

Reply:
Yes, I saw that. Is this a case of updating it again once confluent do support it?

Reply:
This will do for now, but we should have something in the backlog to revisit it. (I'll check with the Confluent folks what the plans are for a 7.6.)

Reply:
FYI: https://r3-cev.atlassian.net/jira/software/c/projects/ES/boards/960?selectedIssue=ES-1801